Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM quay.io/giantswarm/alpine:3.12 AS binaries
+FROM quay.io/giantswarm/alpine:3.14 AS binaries
 
-ARG KUBECTL_VERSION=1.18.9
+ARG KUBECTL_VERSION=1.29.2
```

Comment:
I am wondering why this works at all.. kubectl version should be aligned with kubernetes version.

The url https://storage.googleapis.com/kubernetes-release/release/v1.29.2/bin/linux/amd64/kubectl results in a 404 respponse

Reply:
Oops! Good we have reviews. I read this as the kubectl-gs version.